hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/context/properties/BootstrapProperties.java

```diff
@@ -817,5 +819,6 @@ public static Function<String, Object> transformFor(final String prop) {
             entry(VIRTUALDATASOURCE_JASPERDB_TO_MERKLEDB, AS_BOOLEAN),
             entry(ACCOUNTS_BLOCKLIST_ENABLED, AS_BOOLEAN),
             entry(ACCOUNTS_BLOCKLIST_RESOURCE, AS_STRING),
-            entry(CACHE_CRYPTO_TRANSFER_WARM_THREADS, AS_INT));
+            entry(CACHE_CRYPTO_TRANSFER_WARM_THREADS, AS_INT),
+            entry(CONFIG_VERSION, AS_INT));
```

Comment:
This property may make more sense in the `GLOBAL_STATIC_PROPS` set? (Although as written will still provide the desired functionality.)

Reply:
It is already added to `GLOBAL_STATIC_PROPS` set. As discussed removed from `GlobalDynamicProperties`